src/Effects/Effects.Android/Effects/Entry/EntryClear.cs

```diff
@@ -5,11 +5,11 @@
 using Xamarin.Forms;
 using Xamarin.Forms.Platform.Android;
 
-[assembly: ExportEffect(typeof(ClearEntryEffect), nameof(ClearEntryEffect))]
+[assembly: ExportEffect(typeof(EntryClear), nameof(EntryClear))]
 namespace FormsCommunityToolkit.Effects.Droid
 {
     [Preserve(AllMembers = true)]
-    public class ClearEntryEffect : PlatformEffect
+    public class EntryClear : PlatformEffect
```

Comment:
Should we remove empty overrides? Or leave them as placeholder for future extension (YAGNI ?)